src/Libraries/CoreNodeModelsWpf/NodeViewCustomizations/Watch.cs

```diff
@@ -40,12 +40,12 @@ public void CustomizeView(Watch nodeModel, NodeView nodeView)
 
             var watchTree = new WatchTree();
 
+            watchTree.SetWatchNodeProperties();
+
             // make empty watchViewModel
             rootWatchViewModel = new WatchViewModel(dynamoViewModel.BackgroundPreviewViewModel.AddLabelForPath);
 
             // Fix the maximum width/height of watch node.
-            nodeView.PresentationGrid.MaxWidth = Configurations.MaxWatchNodeWidth;
```

Comment:
Should we also obsolete these two properties? Seems the only references are in this file here which will be removed. 

Or we can still set a much bigger maximum height and width, like half window size or something. This might be a UX question. FYI: @Amoursol If we choose to give user total control, then I prefer these two obsolete now and removed in Dynamo 3.0

Reply:
I discussed this with UX-Jingyi about if we should make a max size of watch node still. Jingyi did not feel that we need. So let's obsolete `MaxWatchNodeWidth` and `MaxWatchNodeHeight` and move on.

Reply:
@QilongTang I'm cool with what you and Jingyi came up with!

Reply:
I think the "watch node" from Grasshopper can be a very good reference. 
Imagine in Dynamo, after double click, we got a Panel, you can write number, string, or any code you want. If you don't write anything, it is a "watch node". With a default size that can display correctly for most of the time. For the special case, you can resize it very easily. Also, you can change the color of the panel.
![image](https://user-images.githubusercontent.com/70321029/107659176-27db4b80-6c87-11eb-8721-12cbaaaabb78.png)